03_scenario.adoc

```diff
@@ -234,23 +234,34 @@ capability is called `my-capability`, its associated secret will be named `my-ca
 
 === Push the Components
 
-TODO: Explain a little what is a component and what will happen when we will do the push
+A component represents a micro-service to be deployed. The Component custom resource provides the descriptor that allows Kubernetes/OpenShift
+to deploy the micro-service and, optionally, expose it outside of the cluster.
+To deploy a microservice two actions need to be done: create the component and push the uber java file.
 
 Before to push the project and to deploy the applications as components, check if you are using your project
 ----
 oc project
 Using project "rhtelab" on server "https://api.cluster-416e.416e.sandbox352.opentlc.com:6443".
 ----
 
-Next deploy the components
+Next create the components
```

Comment:
Why did you decide to split `create` and `push`? Unless there's a specific reason, it's simpler to just do the `push`.

Reply:
Can you answer @aureamunoz ?

Reply:
I don't know anymore, may be wanted to show all possibilities? I looked at the code but I didn't see that `push` creates the component if needed (I've just sawn). I will let the `push`.